--- a/share/kak/rc/filetype/c-family.kak	2024-03-14 22:33:15.559532559 +0900
+++ b/share/kak/rc/filetype/c-family.kak	2024-03-14 22:34:10.809115210 +0900
@@ -287,7 +287,7 @@
         add-highlighter shared/c/code/keywords regex \b($(join "${keywords}" '|'))\b 0:keyword
         add-highlighter shared/c/code/attributes regex \b($(join "${attributes}" '|'))\b 0:attribute
         add-highlighter shared/c/code/types regex \b($(join "${types}" '|'))\b 0:type
-        add-highlighter shared/c/code/values regex \b($(join "${macros}" '|'))\b 0:value
+        add-highlighter shared/c/code/values regex \b($(join "${macros}" '|'))\b 0:macros
     "
 }
 
