--- a/share/kak/rc/filetype/c-family.kak	2024-03-14 22:35:35.638474424 +0900
+++ b/share/kak/rc/filetype/c-family.kak	2024-03-14 22:35:18.418604494 +0900
@@ -223,7 +223,7 @@
 }
 
 # c specific
-add-highlighter shared/c/code/numbers regex %{\b-?(0[xX][0-9a-fA-F]+|\d+)([uU][lL]{0,2}|[lL]{1,2}[uU]?|[fFdDiI]|([eE][-+]?\d+))?|'((\\.)?|[^'\\])'} 0:value
+add-highlighter shared/c/code/numbers regex %{\b-?(0[xX][0-9a-fA-F]+|\d+)([uU][lL]{0,2}|[lL]{1,2}[uU]?|[fFdDiI]|([eE][-+]?\d+))?|'((\\.)?|[^'\\])'} 0:numbers
 evaluate-commands %sh{
     # Grammar
     keywords='asm break case continue default do else for goto if return
