--- a/browser/components/BrowserGlue.sys.mjs	2020-07-19 16:48:37.103020293 +0900
+++ b/browser/components/BrowserGlue.sys.mjs	2020-07-19 17:03:24.935728499 +0900
# https://hg.mozilla.org/mozilla-central/file/tip/browser/components/BrowserGlue.sys.mjs
@@ -74,7 +74,6 @@
   ResetPBMPanel: "resource:///modules/ResetPBMPanel.sys.mjs",
   SafeBrowsing: "resource://gre/modules/SafeBrowsing.sys.mjs",
   Sanitizer: "resource:///modules/Sanitizer.sys.mjs",
-  SaveToPocket: "chrome://pocket/content/SaveToPocket.sys.mjs",
   ScreenshotsUtils: "resource:///modules/ScreenshotsUtils.sys.mjs",
   SearchSERPDomainToCategoriesMap:
     "resource:///modules/SearchSERPTelemetry.sys.mjs",
@@ -1341,7 +1340,7 @@
       lazy.Normandy.init();
     }
 
-    lazy.SaveToPocket.init();
+    // lazy.SaveToPocket.init();
 
     lazy.ResetPBMPanel.init();
 
# from librewolf: disable-pocket
diff --git a/browser/base/content/browser.js b/browser/base/content/browser.js
index 16168e5..857a968 100644
--- a/browser/base/content/browser.js
+++ b/browser/base/content/browser.js
# there is also "SaveToPocket: "chrome://pocket/content/SaveToPocket.jsm","
# need i it?
@@ -5246,8 +5246,6 @@ var XULBrowserWindow = {
 
     SafeBrowsingNotificationBox.onLocationChange(aLocationURI);
 
-    SaveToPocket.onLocationChange(window);
-
     let originalURI;
     if (aRequest instanceof Ci.nsIChannel) {
       originalURI = aRequest.originalURI;
     }

