--- a/browser/components/moz.build	2024-08-05 00:51:41.423030997 +0900
+++ b/browser/components/moz.build	2024-08-05 00:54:56.911577086 +0900
# https://hg.mozilla.org/mozilla-central/file/tip/browser/components/moz.build
# TODO not checked last diff addition. But still don't think that this shit will work.

// that one fixed by second diff
Uncaught Error opening input stream (invalid filename?): chrome://browser/content/shopping/shopping-sidebar.js browser.xhtml:124:25
    <anonymous> chrome://browser/content/browser.xhtml:124

// that one tried to fix at third diff (unchecked)
Uncaught Error: Failed to load resource:///actors/ShoppingSidebarParent.sys.mjs
    _delayedStartup chrome://browser/content/browser-init.js:607
browser-init.js:607:5
    _delayedStartup chrome://browser/content/browser-init.js:607

// nothing in code actually, hope, also fixed by third diff (unchecked)
// Had shown only after sometime in -jsconcole (i guess)
Error: Failed to load resource:///actors/ShoppingSidebarParent.sys.mjs
    onLocationChange chrome://browser/content/browser.js:4334
    callListeners chrome://browser/content/tabbrowser/tabbrowser.js:1027
    _callProgressListeners chrome://browser/content/tabbrowser/tabbrowser.js:1047
    _callProgressListeners chrome://browser/content/tabbrowser/tabbrowser.js:6735
    onLocationChange chrome://browser/content/tabbrowser/tabbrowser.js:7211
tabbrowser.js:1032:23
// TODO there is also: browser/components/BrowserGlue.sys.mjs with "shopping" includes
@@ -61,7 +61,8 @@
     "search",
     "sessionstore",
     "shell",
-    "shopping",
+    # with commenting out just this, browser's ui will break (no tabs)
+    # "shopping",
     "sidebar",
     "syncedtabs",
     "tabbrowser",
--- a/browser/base/content/browser.xhtml	2024-08-05 00:42:51.026975647 +0900
+++ b/browser/base/content/browser.xhtml	2024-08-05 00:48:46.194334208 +0900
# https://hg.mozilla.org/mozilla-central/file/tip/browser/base/content/browser.xhtml
@@ -125,7 +125,7 @@
   Services.scriptloader.loadSubScript("chrome://browser/content/places/places-menupopup.js", this);
   Services.scriptloader.loadSubScript("chrome://browser/content/search/autocomplete-popup.js", this);
   Services.scriptloader.loadSubScript("chrome://browser/content/search/searchbar.js", this);
-  Services.scriptloader.loadSubScript("chrome://browser/content/shopping/shopping-sidebar.js", this);
+  /* Services.scriptloader.loadSubScript("chrome://browser/content/shopping/shopping-sidebar.js", this); */
 
   window.onload = gBrowserInit.onLoad.bind(gBrowserInit);
   window.onunload = gBrowserInit.onUnload.bind(gBrowserInit);
--- a/browser/base/content/browser-init.js	2024-08-05 09:08:55.338467998 +0900
+++ b/browser/base/content/browser-init.js	2024-08-05 09:19:14.233955647 +0900
# https://hg.mozilla.org/mozilla-central/file/tip/browser/base/content/browser-init.js
@@ -606,7 +606,7 @@
 
     CaptivePortalWatcher.delayedStartup();
 
-    ShoppingSidebarManager.ensureInitialized();
+    // ShoppingSidebarManager.ensureInitialized();
 
     SessionStore.promiseAllWindowsRestored.then(() => {
       this._schedulePerWindowIdleTasks();
