--- a/browser/components/moz.build	2024-08-05 00:51:41.423030997 +0900
+++ b/browser/components/moz.build	2024-08-06 00:48:22.920752024 +0900
# https://hg.mozilla.org/mozilla-central/file/tip/browser/components/moz.build
@@ -49,7 +49,7 @@
     "originattributes",
     "pagedata",
     "places",
-    "pocket",
+    # "pocket",
     "preferences",
     "privatebrowsing",
     "prompts",
@@ -67,7 +67,7 @@
     "tabbrowser",
     "tabunloader",
     "textrecognition",
-    "topsites",
+    # "topsites",
     "translations",
     "uitour",
     "urlbar",
