--- a/browser/components/BrowserGlue.sys.mjs	2025-01-08 23:25:17.154016547 +0900
+++ b/browser/components/BrowserGlue.sys.mjs	2025-01-08 23:25:56.983688377 +0900
# https://hg.mozilla.org/mozilla-central/file/tip/browser/components/BrowserGlue.sys.mjs
# just some browser console error about absence of
# NS_ERROR_NOT_IMPLEMENTED: Component returned failure code: 0x80004001 (NS_ERROR_NOT_IMPLEMENTED) [nsIAppStartup.secondsSinceLastOSRestart]
@@ -1937,7 +1937,7 @@
       Glean.startup.isCold.set(isColdStartup);
       Glean.startup.secondsSinceLastOsRestart.set(secondsSinceLastOSRestart);
     } catch (ex) {
-      console.error(ex);
+      // console.error(ex);
     }
   },
 
