--- a/browser/components/BrowserGlue.sys.mjs	2025-01-08 23:25:17.154016547 +0900
+++ b/browser/components/BrowserGlue.sys.mjs	2025-01-09 10:03:24.727427277 +0900
# https://hg.mozilla.org/mozilla-central/file/tip/browser/components/BrowserGlue.sys.mjs
@@ -92,7 +92,7 @@
   SafeBrowsing: "resource://gre/modules/SafeBrowsing.sys.mjs",
   Sanitizer: "resource:///modules/Sanitizer.sys.mjs",
   SandboxUtils: "resource://gre/modules/SandboxUtils.sys.mjs",
-  SaveToPocket: "chrome://pocket/content/SaveToPocket.sys.mjs",
+  // SaveToPocket: "chrome://pocket/content/SaveToPocket.sys.mjs",
   ScreenshotsUtils: "resource:///modules/ScreenshotsUtils.sys.mjs",
   SearchSERPCategorization: "resource:///modules/SearchSERPTelemetry.sys.mjs",
   SearchSERPTelemetry: "resource:///modules/SearchSERPTelemetry.sys.mjs",
@@ -1585,7 +1585,7 @@
       lazy.Normandy.init();
     }
 
-    lazy.SaveToPocket.init();
+    // lazy.SaveToPocket.init();
 
     lazy.ResetPBMPanel.init();
 
# based upon: librewolf: disable-pocket
diff --git a/browser/base/content/browser.js b/browser/base/content/browser.js
--- a/browser/base/content/browser.js	2025-01-09 10:07:43.035334460 +0900
+++ b/browser/base/content/browser.js	2025-01-09 10:12:09.463175892 +0900
# there is also "SaveToPocket: "chrome://pocket/content/SaveToPocket.jsm","
# need i it?
@@ -58,7 +58,7 @@
   PlacesTransactions: "resource://gre/modules/PlacesTransactions.sys.mjs",
   PlacesUIUtils: "resource:///modules/PlacesUIUtils.sys.mjs",
   PlacesUtils: "resource://gre/modules/PlacesUtils.sys.mjs",
-  Pocket: "chrome://pocket/content/Pocket.sys.mjs",
+  // Pocket: "chrome://pocket/content/Pocket.sys.mjs",
   PopupBlockerObserver: "resource:///modules/PopupBlockerObserver.sys.mjs",
   PrivateBrowsingUtils: "resource://gre/modules/PrivateBrowsingUtils.sys.mjs",
   ProcessHangMonitor: "resource:///modules/ProcessHangMonitor.sys.mjs",
@@ -67,7 +67,7 @@
   ResetPBMPanel: "resource:///modules/ResetPBMPanel.sys.mjs",
   SafeBrowsing: "resource://gre/modules/SafeBrowsing.sys.mjs",
   Sanitizer: "resource:///modules/Sanitizer.sys.mjs",
-  SaveToPocket: "chrome://pocket/content/SaveToPocket.sys.mjs",
+  // SaveToPocket: "chrome://pocket/content/SaveToPocket.sys.mjs",
   ScreenshotsUtils: "resource:///modules/ScreenshotsUtils.sys.mjs",
   SearchUIUtils: "resource:///modules/SearchUIUtils.sys.mjs",
   SessionStartup: "resource:///modules/sessionstore/SessionStartup.sys.mjs",
@@ -264,11 +264,11 @@
   "gGfxUtils",
   "chrome://browser/content/browser-graphics-utils.js"
 );
-XPCOMUtils.defineLazyScriptGetter(
-  this,
-  "pktUI",
-  "chrome://pocket/content/pktUI.js"
-);
+// XPCOMUtils.defineLazyScriptGetter(
+//   this,
+//   "pktUI",
+//   "chrome://pocket/content/pktUI.js"
+// );
 XPCOMUtils.defineLazyScriptGetter(
   this,
   "ToolbarKeyboardNavigator",
@@ -3399,7 +3399,7 @@
 
     SafeBrowsingNotificationBox.onLocationChange(aLocationURI);
 
-    SaveToPocket.onLocationChange(window);
+    // SaveToPocket.onLocationChange(window);
 
     UrlbarProviderSearchTips.onLocationChange(
       window,
--- a/browser/base/content/nsContextMenu.sys.mjs	2025-01-10 04:42:46.266614618 +0900
+++ b/browser/base/content/nsContextMenu.sys.mjs	2025-01-10 18:39:58.750211449 +0900
# https://hg.mozilla.org/mozilla-central/file/tip/browser/base/content/nsContextMenu.sys.mjs
# to make context menu working and not only shwoing:
# "gContextMenu is null" and "aPopup is null" (showHideSeparators)
@@ -339,7 +339,7 @@
     this.initViewItems();
     this.initImageItems();
     this.initMiscItems();
-    this.initPocketItems();
+    // this.initPocketItems();
     this.initSpellingItems();
     this.initSaveItems();
     this.initSyncItems();
@@ -889,51 +889,51 @@
     );
   }
 
-  initPocketItems() {
-    const pocketEnabled = Services.prefs.getBoolPref(
-      "extensions.pocket.enabled"
-    );
-    let showSaveCurrentPageToPocket = false;
-    let showSaveLinkToPocket = false;
-
-    // We can skip all this is Pocket is not enabled.
-    if (pocketEnabled) {
-      let targetURL, targetURI;
-      // If the context menu is opened over a link, we target the link,
-      // if not, we target the page.
-      if (this.onLink) {
-        targetURL = this.linkURL;
-        // linkURI may be null if the URL is invalid.
-        targetURI = this.linkURI;
-      } else {
-        targetURL = this.browser?.currentURI?.spec;
-        targetURI = Services.io.newURI(targetURL);
-      }
-
-      const canPocket =
-        targetURI?.schemeIs("http") ||
-        targetURI?.schemeIs("https") ||
-        (targetURI?.schemeIs("about") &&
-          lazy.ReaderMode?.getOriginalUrl(targetURL));
-
-      // If the target is valid, decide which menu item to enable.
-      if (canPocket) {
-        showSaveLinkToPocket = this.onLink;
-        showSaveCurrentPageToPocket = !(
-          this.onTextInput ||
-          this.onLink ||
-          this.isContentSelected ||
-          this.onImage ||
-          this.onCanvas ||
-          this.onVideo ||
-          this.onAudio
-        );
-      }
-    }
-
-    this.showItem("context-pocket", showSaveCurrentPageToPocket);
-    this.showItem("context-savelinktopocket", showSaveLinkToPocket);
-  }
+  // initPocketItems() {
+  //   const pocketEnabled = Services.prefs.getBoolPref(
+  //     "extensions.pocket.enabled"
+  //   );
+  //   let showSaveCurrentPageToPocket = false;
+  //   let showSaveLinkToPocket = false;
+
+  //   // We can skip all this is Pocket is not enabled.
+  //   if (pocketEnabled) {
+  //     let targetURL, targetURI;
+  //     // If the context menu is opened over a link, we target the link,
+  //     // if not, we target the page.
+  //     if (this.onLink) {
+  //       targetURL = this.linkURL;
+  //       // linkURI may be null if the URL is invalid.
+  //       targetURI = this.linkURI;
+  //     } else {
+  //       targetURL = this.browser?.currentURI?.spec;
+  //       targetURI = Services.io.newURI(targetURL);
+  //     }
+
+  //     const canPocket =
+  //       targetURI?.schemeIs("http") ||
+  //       targetURI?.schemeIs("https") ||
+  //       (targetURI?.schemeIs("about") &&
+  //         lazy.ReaderMode?.getOriginalUrl(targetURL));
+
+  //     // If the target is valid, decide which menu item to enable.
+  //     if (canPocket) {
+  //       showSaveLinkToPocket = this.onLink;
+  //       showSaveCurrentPageToPocket = !(
+  //         this.onTextInput ||
+  //         this.onLink ||
+  //         this.isContentSelected ||
+  //         this.onImage ||
+  //         this.onCanvas ||
+  //         this.onVideo ||
+  //         this.onAudio
+  //       );
+  //     }
+  //   }
+
+  //   this.showItem("context-pocket", showSaveCurrentPageToPocket);
+  //   this.showItem("context-savelinktopocket", showSaveLinkToPocket);
+  // }
 
   initSpellingItems() {
     let { document } = this;
--- a/browser/base/content/browser-context.js	2025-01-10 18:49:20.425664181 +0900
+++ a/browser/base/content/browser-context.js	2025-01-10 18:50:39.305025583 +0900
# https://hg.mozilla.org/mozilla-central/file/tip/browser/base/content/browser-context.js
@@ -57,9 +57,9 @@
         case "context-savelink":
           gContextMenu.saveLink();
           break;
-        case "context-savelinktopocket":
-          Pocket.savePage(gContextMenu.browser, gContextMenu.linkURL);
-          break;
+        // case "context-savelinktopocket":
+        //   Pocket.savePage(gContextMenu.browser, gContextMenu.linkURL);
+        //   break;
         case "context-copyemail":
           gContextMenu.copyEmail();
           break;
@@ -163,13 +163,13 @@
         case "context-savepage":
           gContextMenu.savePageAs();
           break;
-        case "context-pocket":
-          Pocket.savePage(
-            gContextMenu.browser,
-            gContextMenu.browser.currentURI.spec,
-            gContextMenu.browser.contentTitle
-          );
-          break;
+        // case "context-pocket":
+        //   Pocket.savePage(
+        //     gContextMenu.browser,
+        //     gContextMenu.browser.currentURI.spec,
+        //     gContextMenu.browser.contentTitle
+        //   );
+        //   break;
         case "fill-login-generated-password":
           gContextMenu.useGeneratedPassword();
           break;
