--- a/browser/base/content/nsContextMenu.sys.mjs	2025-01-10 04:42:46.266614618 +0900
+++ b/browser/base/content/nsContextMenu.sys.mjs	2025-01-10 04:43:30.916236985 +0900
# XXX also in /browser/components/moz.build
# https://hg.mozilla.org/mozilla-central/file/tip/browser/base/content/nsContextMenu.sys.mjs
# To unbreak RMB context menu on all occurences tried this, but
# as 'nsContextMenu.sys.mjs' has now disabled 'buildAskChatMenu', so element (menu#context-ask-chat) has no 'menupopup',
# so raise error in building menus, so clicking on menu item do nothing ( this.showHideSeparators(menuItem.menupopup) )
# All in all, had to disable disabling genai.
@@ -13,7 +13,7 @@
     "resource://gre/modules/ContextualIdentityService.sys.mjs",
   DevToolsShim: "chrome://devtools-startup/content/DevToolsShim.sys.mjs",
   E10SUtils: "resource://gre/modules/E10SUtils.sys.mjs",
-  GenAI: "resource:///modules/GenAI.sys.mjs",
+  // GenAI: "resource:///modules/GenAI.sys.mjs",
   LoginHelper: "resource://gre/modules/LoginHelper.sys.mjs",
   LoginManagerContextMenu:
     "resource://gre/modules/LoginManagerContextMenu.sys.mjs",
@@ -855,10 +855,10 @@
 
     this.showAndFormatSearchContextItem();
     this.showTranslateSelectionItem();
-    lazy.GenAI.buildAskChatMenu(
-      document.getElementById("context-ask-chat"),
-      this
-    );
+    // lazy.GenAI.buildAskChatMenu(
+    //   document.getElementById("context-ask-chat"),
+    //   this
+    // );
 
     // srcdoc cannot be opened separately due to concerns about web
     // content with about:srcdoc in location bar masquerading as trusted

--- a/browser/components/BrowserGlue.sys.mjs	2025-01-08 23:25:17.154016547 +0900
+++ b/browser/components/BrowserGlue.sys.mjs	2025-01-10 04:58:13.099149529 +0900
# https://hg.mozilla.org/mozilla-central/file/tip/browser/components/BrowserGlue.sys.mjs
@@ -53,7 +53,7 @@
   FilePickerCrashed: "resource:///modules/FilePickerCrashed.sys.mjs",
   FormAutofillUtils: "resource://gre/modules/shared/FormAutofillUtils.sys.mjs",
   FxAccounts: "resource://gre/modules/FxAccounts.sys.mjs",
-  GenAI: "resource:///modules/GenAI.sys.mjs",
+  // GenAI: "resource:///modules/GenAI.sys.mjs",
   HomePage: "resource:///modules/HomePage.sys.mjs",
   Integration: "resource://gre/modules/Integration.sys.mjs",
   Interactions: "resource:///modules/Interactions.sys.mjs",
@@ -638,22 +638,22 @@
     allFrames: true,
   },
 
-  GenAI: {
-    parent: {
-      esModuleURI: "resource:///actors/GenAIParent.sys.mjs",
-    },
-    child: {
-      esModuleURI: "resource:///actors/GenAIChild.sys.mjs",
-      events: {
-        DOMContentLoaded: {},
-        mousemove: {},
-        resize: {},
-        scroll: {},
-      },
-    },
-    allFrames: true,
-    enablePreference: "browser.ml.chat.enabled",
-  },
+  // GenAI: {
+  //   parent: {
+  //     esModuleURI: "resource:///actors/GenAIParent.sys.mjs",
+  //   },
+  //   child: {
+  //     esModuleURI: "resource:///actors/GenAIChild.sys.mjs",
+  //     events: {
+  //       DOMContentLoaded: {},
+  //       mousemove: {},
+  //       resize: {},
+  //       scroll: {},
+  //     },
+  //   },
+  //   allFrames: true,
+  //   enablePreference: "browser.ml.chat.enabled",
+  // },
 
   LightweightTheme: {
     child: {
@@ -3150,12 +3150,12 @@
         },
       },
 
-      {
-        name: "GenAI.init",
-        task() {
-          lazy.GenAI.init();
-        },
-      },
+      // {
+      //   name: "GenAI.init",
+      //   task() {
+      //     lazy.GenAI.init();
+      //   },
+      // },
 
       {
         name: "QuickSuggest.init",
