--- a/browser/components/moz.build	2025-10-20 02:27:50.805946188 +0900
+++ b/browser/components/moz.build	2025-10-20 02:29:33.455183256 +0900
# https://hg.mozilla.org/mozilla-central/file/tip/browser/components/moz.build
@@ -39,6 +39,8 @@
     "downloads",
     "enterprisepolicies",
     "extensions",
+    # commenting it out breaks browser (ui) (including opened and opening of tabs)
+    # or not, need someday to check ('cause it should be because of shopping, but who knows). Got also firefoxview icon around tabbar with it commented out, and not such thing without it, so could be
     "firefoxview",
     "genai",
     "ipprotection",
@@ -57,6 +59,7 @@
     "protocolhandler",
     "reportbrokensite",
     "resistfingerprinting",
+    # never disable this thing (needed for tabs to work properly)
     "screenshots",
     "search",
     "security",
@@ -67,8 +70,8 @@
     "tabbrowser",
     "tabunloader",
     "taskbartabs",
-    "textrecognition",
-    "topsites",
+    # "textrecognition",
+    # "topsites",
     "translations",
     "uitour",
     "urlbar",
